features/step_definitions/block-consolidation-page-steps.js

```diff
@@ -0,0 +1,92 @@
+import { When, Then } from 'cucumber';
+import { expect } from 'chai';
+
+const BLOCK_CONSOLIDATION_COMPONENT = '.BlockConsolidationStatus_component';
+const BLOCK_CONSOLIDATION_EXPLANATION = '.BlockConsolidationStatus_content p:nth-child(3)';
+const EPOCHS_CONSOLIDATION_STATUS = '.BlockConsolidationStatus_epochs p span';
+const EPOCHS_CONSOLIDATED = '.BlockConsolidationStatus_indicatorEpochsConsolidated p';
+const TRAILING_BY_2_EPOCH = '.BlockConsolidationStatus_indicatorEpochsBehind p';
+const MAXIMUM_EPOCH = '.BlockConsolidationStatus_fullEpoch';
+const SYNC_PROGRESS = '.BlockConsolidationStatus_indicatorEpochsSynced p span';
+
+When(/^I toggle the Block Consolidation Status Page$/, async function () {
+  await this.client.execute(() => daedalus.actions.app.toggleBlockConsolidationStatusScreen.trigger());
+});
+
+Then(/^the Block Consolidation Status Page is (hidden|visible)/, async function (state) {
+  const isVisible = state === 'visible';
+  return await this.client.waitForVisible(BLOCK_CONSOLIDATION_COMPONENT, null, !isVisible);
+});
+
+Then(/^the page accurately renders an explanation of how block consolidation works in file storage$/, async function () {
+  await this.client.waitForText(BLOCK_CONSOLIDATION_EXPLANATION);
+  const explanationText = await this.client.getText(BLOCK_CONSOLIDATION_EXPLANATION);
```

Comment:
You can simplify the those two lines by using the helper `this.getVisibleTextsForSelector(this.client, BLOCK_CONSOLIDATION_EXPLANATION)` (same for all other occasions of `waitForText` + `getText`)